.github/workflows/ci-pipeline.yaml

```diff
@@ -28,6 +28,9 @@ jobs:
       - name: Test and coverage
         run: yarn jest --coverage
 
+      - name: Check types
+        run: yarn tsc --noEmit
```

Comment:
Yes, good plan.

Reply:
although it is currently being an arse because it's complaining about octokit types

Reply:
in node_modules 😠